### DISCUSSION TITLE
Accidently remove my own team with no member

### DISCUSSION THREAD

**Discussion by irwansyafani** [vercel/community#6931] [Account Issue] [answered]
### Question

Hi There,

I accidently remove my own team (_**irwansyafanis-projects**_). Can Vercel team help me to recover it?

**Comment by irwansyafani** [comment]
I've read [this post](https://github.com/orgs/vercel/discussions/4120), but just in case Vercel team can help :)

**Reply by amyegan** [reply, marked as answer]
Hey @irwansyafani. That post is still correct. We can't recover deleted teams. 🙁 

You can create new team and new projects to replace the ones deleted with the original team. If you need help recovering a custom domain lost when the team was deleted, that's something [the Customer Success team can help](https://vercel.com/help) with.
